src/components/Search/Search.component.jsx

```diff
@@ -2,9 +2,20 @@ import React from 'react';
 import { Icon, SearchContainer, IconContainer, InputContainer, Input } from './styled';
 import icons from '../../icon-svg.json';
 import { useVideos } from '../../providers/Context';
+import { useHistory } from 'react-router-dom';
+import { youtube } from '../../api/youtube';
 
 function Search(props) {
-  const updateVideo = useVideos().searchVideos;
+  //const updateVideo = useVideos().searchVideos;
+
+  let history = useHistory();
+
+  function handleSearch(e) {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      history.push(`/search/${e.target.value}`);
```

Comment:
You can program defensively like:
```suggestion
  function handleSearch({ key, preventDefault = () => {}, target: { value } = {}} = {}) {
    if (key === 'Enter') {
      preventDefault();
      history.push(`/search/${value}`);
```